README.md

```diff
@@ -156,10 +156,7 @@ module.exports = class Item extends Model {
       options: {
         classMethods: {
           associate: (models) => {
-            models.Item.belongsToMany(models.User, {
```

Comment:
Maybe we should let this way if users want more "granularity" about relations, because your solution is nice but maybe in some cases (don't have any in mind) user will want to manage his relations by himself to present some limitation we can have about this. What do you think ?